test/js/web-animations-next-animation.js

```diff
@@ -0,0 +1,64 @@
+suite('web-animations-next-animation-tests', function() {
```

Comment:
Do you want to test cases where a Promise is recreated, ala the `_reset` methods?